### ISSUE TITLE
Doesn't work game with bots when playing offline

### ISSUE THREAD

**Issue by DXN41K** [ValveSoftware/Dota2-Gameplay#15577]
The game doesn't start a game with bots, when I'm offline. When I'm starting the game, there shows splash screen, and then it returns me in main menu

**Comment by vlady0415** [comment]
Yes, I already tried to play the vs bot offline and it's not working. I hope they will fix it soon.
